src/index.html

```diff
@@ -7,8 +7,53 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div>
+        <img
+        src="/images/logo.png"
+        alt="logo"
+        class="logo"
+        >
+      </div>
+      <nav class="nav">
+        <ul class="ul">
+          <li>
+            <a href="#apple" class="link is-active apple">APPLE</a>
+          </li>
```

Comment:
It is necessary to return the name of BEM classes
```suggestion
        <ul class="nav__list">
          <li class="nav__item">
            <a href="#apple" class="nav__link is-active">APPLE</a>
          </li>
```